htdocs/margin/customerMargins.php

```diff
@@ -220,6 +220,7 @@
 if (!$user->rights->societe->client->voir && ! $socid) $sql.= " AND s.rowid = sc.fk_soc AND sc.fk_user = " .$user->id;
 $sql.= " AND f.fk_statut NOT IN (" . implode(', ', $invoice_status_except_list) . ")";
 $sql.= ' AND s.entity IN ('.getEntity('societe').')';
+$sql.= ' AND f.entity IN ('.getEntity('').')';
```

Comment:
For object of type invoice, the parameter in getEntity must be 'facture' or 'invoice'